### PR TITLE
Add single database configuration instructions

### PR DESCRIPTION
**Pull request by jimmypoulsen**
I'm running a super simple application that used Sidekiq and Redis. I replaced that with Solid Queue and Solid Cable. However, because the application is super slim I don't think I need to separate the queue and cable into their own databases. I found the instructions for doing so in the Solid Queue repository but couldn't find it for Solid Cable.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -52,6 +52,16 @@ production:
 
 Then run `db:prepare` in production to ensure the database is created and the schema is loaded.
 
+### Single database configuration
+
+Running Solid Cable in a separate database is recommended, but it's also possible to use one single database for both the app and the cable. Just follow these steps:
```
Comment (npezza93) [inline README.md:57]:
```suggestion
Running Solid Cable in a separate database is recommended, but it's also possible to use a single database for both the app and the cable.
```

File: README.md [hunk 2]
```diff
@@ -52,6 +52,16 @@ production:
...
+
+1. Copy the contents of `db/cable_schema.rb` into a normal migration and delete `db/cable_schema.rb`
+2. Remove `connects_to` from `config/cable.yml`
+3. Migrate your database
```
Comment (npezza93) [inline README.md:61]:
```suggestion
3. `bin/rails db:migrate`
```

File: README.md [hunk 3]
```diff
@@ -52,6 +52,16 @@ production:
...
+
+You won't have multiple databases, so `database.yml` doesn't need to have primary and cable database.
```
Comment (npezza93) [inline README.md:63]: Do we need this line? The generator doesnt touch this file so i think we might be able to get away with not having it.

**Review by npezza93 — CHANGES_REQUESTED** [review]

File: README.md (hunk 3 above)
Comment (jimmypoulsen) [reply README.md:63]: I implemented this based on what's in the Solid Queue README. I think it makes sense to keep it here to emphasize that the configuration in `database.yml` should be left out.

**Review by npezza93 — APPROVED** [review]

File: README.md [hunk 4]
```diff
@@ -52,6 +52,16 @@ production:
 
 Then run `db:prepare` in production to ensure the database is created and the schema is loaded.
 
+### Single database configuration
+
+Running Solid Cable in a separate database is recommended, but it's also possible to use a single database for both the app and Action Cable.
+
+1. Copy the contents of `db/cable_schema.rb` into a normal migration and delete `db/cable_schema.rb`
+2. Remove `connects_to` from `config/cable.yml`
```
Comment (rnestler) [inline README.md:60]: Shouldn't we still have a `connects_to` (just pointing to `primary`) to ensure that it uses a separate DB connection?

**Comment by santib** [comment]
Just curious, what are the limitations or the reasons why to NOT use solid_cable with the same DB? Is it because of the amount of connections to the DB? Because of the load it can have on the DB? Something else?

**Comment by npezza93** [comment]
@santib For sqlite, you can only have one concurrent writer at a time to your db. So splitting solid* out to their own dbs allows you to write to the solid databases while not tying up the writer on your main db.

**Comment by santib** [comment]
@npezza93 thanks for the clarification! I was actually thinking on PostgreSQL.. do you have any insights for PostgreSQL/MySQL/similar ?

**Pull request merged**